### ISSUE TITLE
[BUG] IPv6 not respected when creating VNET jails

### ISSUE THREAD

**Issue by hackacad** [BastilleBSD/bastille#361]
**[MANDATORY] Describe the bug [MANDATORY]**
IPv6 addresses are handled as v4 when creating VNET-jails.

```
bastille create -V v6jail 13.0-RC2 fc00:1000:1000::1 em0
Valid: (fc00:1000:1000::1).
Valid: (em0).

NAME: v6jail.
IP: fc00:1000:1000::1.
INTERFACE: em0.
RELEASE: 13.0-RC2.
```

```
[v6jail]:
root@v6jail:~ # cat /etc/rc.conf | grep inet
ifconfig_vnet0="inet fc00:1000:1000::1"
```
should be:
`ifconfig_vnet0_ipv6="inet6 fc00:1000:1000::1"`

**[MANDATORY] Bastille and FreeBSD version (paste ``bastille -v && freebsd-version -kru`` output)**
```
bastille -v && freebsd-version -kru
0.8.20210115
13.0-RC2
13.0-RC2
13.0-RC2
```

**[MANDATORY] How did you install bastille? (port/pkg/git)**
ports

**Comment by noracenofun** [comment]
This issue has been fixed in the following pull request #379

**Comment by cedwards** [comment]
Closing with fix in #379 . Let me know if somehow problem persists.
